tests/vm-prep

```diff
@@ -64,6 +64,12 @@ prepare()
 		$VIRSH net-undefine $NETWORK_NAME
 	fi
 
+    # In case a network has leaked in from elsewhere
+    if ip address show dev $NETWORK_NAME >/dev/null 2>/dev/null; then
+        ip link set $NETWORK_NAME down || true
+        brctl delbr $NETWORK_NAME || true
+    fi
+
```

Comment:
No change actually, but I now see that it's already rather garbled, so it doesn't make much of a difference. :-)